config.json

```diff
@@ -18,6 +18,17 @@
         "demo_param1": "demo_param1",
         "demo_param2": 100
       }
+    },
+    {
+      "casename": "Testcase 01_get_request",
+      "pre_call": "echo 'This is called BEFORE cmdline call'",
+      "post_call": "echo 'This is called AFTER cmdline call'",
+      "stoponfail": "true",
+      "cmdline": "python3.6 testcases/01_get_request.py <account1> <account2> <pubkey>",
```

Comment:
please use the `exist_account` parameters in the test script file, avoid add redundant account parameters